x-pack/plugins/ingest_manager/public/plugin.ts

```diff
@@ -60,13 +60,16 @@ export class IngestManagerPlugin
   implements
     Plugin<IngestManagerSetup, IngestManagerStart, IngestManagerSetupDeps, IngestManagerStartDeps> {
   private config: IngestManagerConfigType;
+  private kibanaVersion: string;
```

Comment:
What do you think about changing this to `env`? It seems similar to what we have in `config`.  

`useKibanaVersion` could be updated to read the EnvContext and other hooks/functions could get branch, or dev vs prod, etc if/when it comes up.

Reply:
Sounds good to me.  I didn't put it in `IngestManagerConfigType` just because those are from the config file, so I think like you said something like "env" sounds good.  I opened a follow up issue https://github.com/elastic/kibana/issues/79483